doc/nrf/external_comp/coremark.rst

```diff
@@ -0,0 +1,88 @@
+.. _integration_template:
+
+CoreMark integration
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+CoreMark is a CPU benchmark provided by EEMBC (Embedded Microprocessor Benchmark Consortium) to verify SoCs performance.
+It provides a series of test done in each benchmark iteration.
+The number of iterations DUT can perform per second gives benchmark score.
+Integrating CoreMark into |NCS| allows users to easily verify performance of acquired SoCs.
+
+
+Integration prerequisites
+*************************
+
+Before you start the |NCS| integration with CoreMark, make sure that the following prerequisites are completed:
+
+* :ref:`Installation of the nRF Connect SDK <installation>`.
+* Setup of the required :term:`Development Kit (DK)`.
+
+Solution architecture
+*********************
+
+CoreMark repository is added by default as a module to |NCS| by adding CoreMark repository to |NCS| west manifest.
+It can be found under path :file:`modules/benchmark/coremark`.
+To run CoreMark correctly SoC needs a porting layer.
+The porting layer for devices supported in |NCS| was added and placed in: :file:`nrf/modules/coremark`.
```

Comment:
```suggestion
CoreMark needs a porting layer to run correctly on an SoC.
The porting layer for devices supported in the |NCS| are under the :file:`nrf/modules/coremark` path.
```

Reply:
"an SoC" ?  Shouldn't it be "a SoC"?

Reply:
I think we've had a similar discussion before with the team regarding the HID report. 
Similarly to that case, it depends on how you read it (each letter separately, starting with 'eS', and that puts the vowel first, or as a one word, in which case the first letter is a consonant). 
So both are correct but we can definitely keep "a" as I believe it's generally more common, so let's leave as is.